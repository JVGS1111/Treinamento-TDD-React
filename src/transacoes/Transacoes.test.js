import React from 'react';
import { render } from '@testing-library/react'
import Transacoes from './Transacoes'

describe('Sobre a lista de transacoes', () => {

    it('O componente exibido corresponde ao seu snapshot', () => {


        const { container } = render(<Transacoes />)

        expect(container.firstChild).toMatchSnapshot();

    })
})