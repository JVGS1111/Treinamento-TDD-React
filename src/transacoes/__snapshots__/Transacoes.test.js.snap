// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sobre a lista de transacoes O componente exibido corresponde ao seu snapshot 1`] = `
<div
  data-testid="transacoes"
/>
`;
